Src/AutoFixture/FreezeOnMatchCustomization.cs

```diff
@@ -29,6 +29,7 @@ public class FreezeOnMatchCustomization : ICustomization
         /// <exception cref="ArgumentNullException">
         /// <paramref name="targetType"/> is <see langword="null"/>.
         /// </exception>
+        [Obsolete("Please use FreezeOnMatchCustomization(object, IRequestSpecification) instead.")]
```

Comment:
The code from the [removed constructor](new ExactTypeSpecification(targetType)):  `new ExactTypeSpecification(targetType)`.
That might be a bit unclear. I'll extend the obsolete message.